README.md

```diff
@@ -239,8 +239,11 @@ Bump new version number in:
 Commit the new version number:
 
 ```sh
+old_version="1.0.1"
```

Comment:
suggestion: it would be nice if we could achieve the same, but without having to specify the old version number. I'm pretty sure it's possible with a regexp like `var Version = ".*"`.